src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml

```diff
@@ -67,33 +66,31 @@
 	
 	<bean id="PurchaseOrder-commodityCodeValidation" parent="PurchaseOrder-commodityCodeValidation-parentBean" scope="prototype" />
 	<bean id="PurchaseOrder-commodityCodeValidation-parentBean" class="edu.cornell.kfs.module.purap.document.validation.impl.PurchaseOrderCommodityCodeValidation" abstract="true">
-		<property name="dataDictionaryService" ref="dataDictionaryService" />
+		<property name="businessObjectDictionaryService" ref="businessObjectDictionaryService" />
 		<property name="businessObjectService" ref="businessObjectService" />	
 		<property name="parameterService" ref="parameterService" />	
 	</bean>
 	
-	<bean id="Purchasing-addItemValidation" class="org.kuali.kfs.module.purap.document.validation.impl.PurchasingAddItemValidation" abstract="true">
-		<property name="dataDictionaryService" ref="dataDictionaryService" />
-		<property name="dictionaryValidationService" ref="dictionaryValidationService" />		
-		<property name="businessObjectService" ref="businessObjectService" />		
-		<property name="commodityCodeValidation" ref="Purchasing-commodityCodeValidation" />
-	</bean>
+	<bean id="Purchasing-addItemValidation"
+		class="org.kuali.kfs.module.purap.document.validation.impl.PurchasingAddItemValidation" abstract="true"
+		p:businessObjectDictionaryService-ref="businessObjectDictionaryService"
+		p:dictionaryValidationService-ref="dictionaryValidationService"
+		p:businessObjectService-ref="businessObjectService"/>
 	
-	<bean id="Purchasing-importItemValidation" class="org.kuali.kfs.module.purap.document.validation.impl.PurchasingImportItemValidation" abstract="true">
-		<property name="dataDictionaryService" ref="dataDictionaryService" />
-		<property name="dictionaryValidationService" ref="dictionaryValidationService" />		
-		<property name="businessObjectService" ref="businessObjectService" />		
-		<property name="commodityCodeValidation" ref="Purchasing-commodityCodeValidation" />
```

Comment:
This commodityCodeValidation reference should be copied over to the revised bean.

Also, I noticed that our overlay of PurchasingImportItemValidation supports some additional properties that do not appear to be in active use (and are not set by the old or new version of the bean above). We should create a user story to clean up those properties from our overlay accordingly.

Reply:
I fixed the reference, and created KFSPTS-18645.